cognite/neat/_graph/extractors/_classic_cdf/_relationships.py

```diff
@@ -67,25 +67,33 @@ def _log_target_nodes_if_set(self, item: Relationship) -> Relationship:
         return item
 
     def _item2triples_special_cases(self, id_: URIRef, dumped: dict[str, Any]) -> list[Triple]:
-        if self.identifier == "externalId":
-            return []
         triples: list[Triple] = []
         if (source_external_id := dumped.pop("sourceExternalId")) and "sourceType" in dumped:
             source_type = dumped["sourceType"]
-            try:
-                source_uri = self._uri_by_external_id_by_by_type[InstanceIdPrefix.from_str(source_type)][
-                    source_external_id
-                ]
-            except KeyError:
-                warnings.warn(
-                    NeatValueWarning(f"Missing externalId {source_external_id} for {source_type}"), stacklevel=2
+            source_prefix = InstanceIdPrefix.from_str(source_type)
+            if self.identifier == "id":
+                try:
+                    source_uri = self._uri_by_external_id_by_type[source_prefix][source_external_id]
+                except KeyError:
+                    warnings.warn(
```

Comment:
Maybe nitpick (I don't know the full context): 
Same here, should be error as well?

Reply:
Again same argument, when extracting we should not raise an error just because a single instance had issues. Instead warn about it an continue.